README.md

```diff
@@ -8,9 +8,22 @@ It contains two parts:
 - A fluentbit daemonset: Small collector running on every node collecting containers and audit logs and for forwarding them to fluentd.
 - A fluentd deployment: Takes care of persisting the logs to any data sink
 
+## Requirements
+
+- You can install only one release of this chart per kubernetes cluster
+- By default any forwarded is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```

Comment:
Is `forward` a thing that can be active? or should it be `forwarded logs are active`? or `any log forwarding is active` 

And what do you mean with `any` ? Should it be `all` ?

```suggestion
- By default any forward is active so make sure you check [configuration document](helm/fluent-logshipping-app/Configuration.md) before deploy it in your cluster.
```

Reply:
I think it should be "no forward" as I had to disable all of them by default when I added Azure support